code-examples/low-latency-sessions/main.go

```diff
@@ -0,0 +1,134 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/gin-gonic/gin"
+	"github.com/go-redis/redis/v8"
+
+	client "github.com/ory/kratos-client-go"
+)
+
+const (
+	// For local development, it's requires to use
+	// ory proxy to make cookies work fine.
```

Comment:
```suggestion
	// When developing locally, use Ory Proxy
	// to have the right origin set in HTTP cookies.
```

Maybe instead of the proxy we could promote the Tunnel here?